modin/backends/pandas/query_compiler.py

```diff
@@ -532,6 +532,45 @@ def is_monotonic_decreasing(self):
 
     # END Reduction operations
 
+    def resample(
+        self,
+        resample_args,
+        func_name,
+        is_ser_out=False,
```

Comment:
Ideally, there query compiler does not have to worry about `Series` vs `DataFrame`. Is this possible?

Reply:
This bool value checks needing of recalculating `new_columns` in `_apply_full_axis` function. In general, this check may be excluded, but `columns` will be recalculated for `Series`-output in this case. I tried this case and get an unexpected result where field `name` of `Series`-output was set by `0` value. This is need to investigate.

Pandas  `Series` for checking:
```
2000-01-01 00:00:00    2
2000-01-01 00:02:00    2
2000-01-01 00:04:00    2
2000-01-01 00:06:00    2
2000-01-01 00:08:00    2
2000-01-01 00:10:00    2
2000-01-01 00:12:00    1
Freq: 2T, dtype: int64
```
Modin `Series`:
```
2000-01-01 00:00:00    2
2000-01-01 00:02:00    2
2000-01-01 00:04:00    2
2000-01-01 00:06:00    2
2000-01-01 00:08:00    2
2000-01-01 00:10:00    2
2000-01-01 00:12:00    1
Freq: 2T, Name: 0, dtype: int64
Name: 0, dtype: int64
```
Also, issue about incorrect output of `modin.Series` was created(#1646 )